### ISSUE TITLE
Configuration issue

### ISSUE THREAD

**Issue by GodsonPattudurai** [play-rconf/play-rconf-git#3]
![Screenshot (31)](https://user-images.githubusercontent.com/26676155/121171385-14767080-c874-11eb-9abf-7f81a1436270.png)

Hi Team, Need a help on attached issue.

I am trying to configure git using below rconf play config,

play.application.loader = "io.playrconf.ApplicationLoaderScala"

remote-configuration {
  provider = "io.playrconf.provider.GitProvider"

  git {
       uri = "https://github.x.com/server/remote-config.git",
       filepath = "/src/main/resources/static/server-dev.conf",
       mode = "user",
       user.login = "user@x.com",
       user.password = "xxxxx"

       #uri = "git@github.x.com:Server/remote-config.git"
       #mode = "ssh-rsa"
       #ssh-rsa.private-key = "./conf/id_rsa"
  }
}

**Comment by thibaultmeyer** [comment]
Hello,

does your repository configuration allow to connect with the chosen credentials? The error reported looks like a server side error...

**Comment by GodsonPattudurai** [comment]
Yes, I can see the user in repository. But issue is while we trying to use the library in github enterprise. Since this has different authentication.

So we tried using ssh-rsa., that also not worked. Could you please help us with some sample example for ssh-rsa or github. It will be more helpful for us.

**Comment by GodsonPattudurai** [comment]
![screencapture-localhost-9000-2021-06-09-12_33_38](https://user-images.githubusercontent.com/26676155/121309093-658f6e80-c91f-11eb-86cf-3d620b422943.png)

Please find the configuration below,

#uri = "git@github.x.com:Server/remote-config.git"
#mode = "ssh-rsa"
#ssh-rsa.private-key = "./conf/id_rsa"

But key is mapped with the user account, we suspecting this may cause using private key path what we mentioned is wrong. It will be more helpfull if we have some working code base.

Thanks,
Godson

**Comment by thibaultmeyer** [comment]
Did you try to use fullpath for the SSH RSA private key ?

**Comment by felipebonezi** [comment]
> ![screencapture-localhost-9000-2021-06-09-12_33_38](https://user-images.githubusercontent.com/26676155/121309093-658f6e80-c91f-11eb-86cf-3d620b422943.png)
> 
> Please find the configuration below,
> 
> #uri = "[git@github.x.com](mailto:git@github.x.com):Server/remote-config.git" #mode = "ssh-rsa" #ssh-rsa.private-key = "./conf/id_rsa"
> 
> But key is mapped with the user account, we suspecting this may cause using private key path what we mentioned is wrong. It will be more helpfull if we have some working code base.
> 
> Thanks, Godson

Do you still need help?
